lib/pdf_fill/forms/va21p527ez.rb

```diff
@@ -1343,7 +1343,7 @@ def expand_veteran_service_information
         @form_data['serviceBranch'] = @form_data['serviceBranch']&.select { |_, value| value == true }
 
         @form_data['pow'] = to_radio_yes_no(@form_data['powDateRange'].present?)
-        if @form_data['pow'].zero?
+        if @form_data['pow'] == 1
```

Comment:
Do you want to do a .to_i here? Are we sure this will always be an integer and not a string?

Reply:
@LindseySaari Yes, this value will always be an int value because we're defining it as an int in our `to_radio_yes_no` function.